### DISCUSSION TITLE
Bulk URL Import with category selection

### DISCUSSION THREAD

**Discussion by DiscusZ** [mealie-recipes/mealie#1704] [Feature Request]
Would love to be able to do bulk import from a file, or text box with recipe URL, with ability to  set category  as well.  Speed up Automation of adding.

I also have hundreds of PDF files I created from URLs  would be nice to be able to import those  :)

**Comment by michael-genson** [comment]
This is now a feature during bulk import; you can set tags/categories to all imported recipes
